playbooks/technology-choices.md

```diff
@@ -6,106 +6,77 @@ description: Evaluating and adopting new technology at Artsy
-  better-suited architecture for near-real-time event handling. It currently exclusively handles processing of live
-  auction events and shadows the Gravity bidding engine for timed auctions. Eventually, it aims to be the
-  source-of-truth for all auction state.
-- **Responsive page designs** over separate mobile designs or implementations. In this way we aim to get the most
-  leverage from the least code and share a design library and components. As more of our usage in general
-  transitions to mobile, it's critical that those visitors have at least parity with the desktop experience.
-- **Mobile software development** is split across two frameworks and three languages: native code is written with
-  UIKit in Objective-C and Swift, while React Native development is done with TypeScript. Our mobile language of
-  choice is Objective-C because core infrastructure of our app, [Eigen](https://github.com/artsy/eigen), is written
-  in it. Additionally, React Native itself is also written in Objective-C. New features in our iOS app should be
-  written using React Native. Our goal when using React Native is to never compromise on software quality for the
-  sake of a cross-platform framework, so sometimes we do need to write native
-  code in Objective-C, using UIKit.
+## Evolving Technology at Artsy
+
+We want to accomplish a lot with a lean team, which means we must
+[choose stable technologies](http://boringtechnology.club/). However, we also want to adopt best-of-breed
+technologies or best-suited tools to a given need, which may need work or still be evolving. We've borrowed from
+[ThoughtWorks' Radar](https://www.thoughtworks.com/radar/faq) to define the following stages for evaluating,
+adopting, and retiring technologies:
+
+- **Adopt:** Reasonable defaults for most work. These choices have been exercised successfully in production at
+  Artsy and there is a critical mass of engineers comfortable working with them.
+- **Trial:** These are being evaluated in limited production circumstances. We don't have enough production
+  experience to recommend them for high-risk or business-critical use cases, but they may be worth consideration if
+  your project seems like a fit.
+- **Assess:** Technologies we are interested in and maybe even built proofs-of-concept for, but haven't yet trialed
+  in production.
+- **Hold:** Based on our experience, these technologies should be avoided. We've found them to be flawed, immature,
+  or simply supplanted by better alternatives. In some cases these remain in legacy production uses, but we should
+  take every opportunity to retire or migrate away.
+
+Artsy's current choices can be edited in [technology_radar/data.csv](/playbooks/technology_radar/data.csv) and
+[viewed in radar format here](https://radar.thoughtworks.com/?sheetId=https%3A%2F%2Fraw.githubusercontent.com%2Fartsy%2Freadme%2Fmaster%2Fplaybooks%2Ftechnology_radar%2Fdata.csv).
```

Comment:
total nitpick, but visiting this link displays

<img width="2318" alt="Screen Shot 2019-12-10 at 2 36 09 PM" src="https://user-images.githubusercontent.com/1561546/70562476-75779800-1b5a-11ea-96fa-2b79822556b0.png">

(notice the `DATA` header).

I wonder if renaming data.csv to artsy_technology_radar.csv would look a bit better / be more shareable.

Reply:
Weird. Not sure if this functionality changed or I just never noticed that before.

I'll rename in a separate PR to keep this PR's diff readable.